pandas/core/accessor.py

```diff
@@ -196,7 +194,7 @@ def decorator(accessor):
     return decorator
 
 
-_doc = """\
+_doc = """
```

Comment:
I'm a little wary of shared docstrings like this as they've been harder to validate in the past. Was this actually throwing an error with the validator and/or have you built some of the affected docstrings locally to make sure they work?

Reply:
This is valid. Just re-checked.

Reply:
Can you post a screenshot of the build of an affected doc? Again want to be extra cautious here as this was what tripped up prior PRs